pkg/actuators/machine/instances.go

```diff
@@ -212,6 +212,13 @@ func getBlockDeviceMappings(blockDeviceMappings []providerconfigv1.BlockDeviceMa
 			Encrypted:  blockDeviceMappings[0].EBS.Encrypted,
 		},
 	}
+
+	if aws.StringValue(blockDeviceMappings[0].EBS.KMSKey.ID) != "" {
```

Comment:
may be include a v(3) log for each branch?

Reply:
hmm.. 
- I don't see any such logging for other AWS References objects?
- Also even if we log this, it can't be tied back the machine as there is no such context available.

but added it to f35cbbff